simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -268,9 +268,10 @@ void doRequest(CollectorRegistry registry, String job, Map<String, String> group
         writer.close();
       }
 
-      int response = connection.getResponseCode();
-      if (response != HttpURLConnection.HTTP_ACCEPTED) {
-        throw new IOException("Response code from " + url + " was " + response);
+      int responseCode = connection.getResponseCode();
+      if (responseCode != HttpURLConnection.HTTP_ACCEPTED) {
+        throw new IOException("Response code from " + url + " was " + responseCode
+            + ", message: " + connection.getResponseMessage());
```

Comment:
Even better, the `getResponseCode` call throws an exception when there's a 400 so I'm not sure it's even possible to get to the content without pulling in dependencies.